src/applications/vaos/services/appointment/transformers.js

```diff
@@ -113,11 +113,10 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruth) {
   const isCC = appt.kind === 'cc';
   const isVideo = appt.kind === 'telehealth' && !!appt.telehealth?.vvsKind;
   const isAtlas = !!appt.telehealth?.atlas;
-  const isPast = isPastAppointment(appt);
-  const isRequest = useFeSourceOfTruth
-    ? appt.pending
-    : appointmentType === APPOINTMENT_TYPES.request ||
-      appointmentType === APPOINTMENT_TYPES.ccRequest;
+  const isPast = useFeSourceOfTruth ? appt.past : isPastAppointment(appt);
+  const isRequest =
```

Comment:
It turns out that this undid part of @jenniemc's changes so we need to revert this part of the change. Not sure why this didn't show up in the [merge commit](https://github.com/department-of-veterans-affairs/vets-website/pull/35147/commits/8e03f67e4a5cd230d7176e039908407306772565) though.